src/views/EventsAttended/index.js

```diff
@@ -8,89 +8,81 @@ import { gordonColors } from 'theme';
 
 import { List, Grid, Button, Typography } from '@material-ui/core';
 
-export default class EventsAttended extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      events: [],
-      loading: true,
-    };
-  }
-  componentDidMount() {
-    if (this.props.authentication) {
-      this.loadEvents();
-    }
-  }
+const EventsAttended = (props) => {
```

Comment:
The only thing you use from props is authentication; perhaps it would be better to only pass in authentication.

Reply:
Good catch!